HIP-1.md

```diff
@@ -0,0 +1,105 @@
+| hip | title               | status | author | created    |
+| --- | ------------------- | ------ | ------ | ---------- |
+| 1   | Sovereign consensus | Draft  | asaj   | 2022-10-04 |
+
+### **Brief Summary / Abstract**
+
+Defines the specification for sovereign consensus and interchain security modules, which allows message recipients to configure their own security models.
+
+### **Motivation**
+
+Interchain security models offer different tradeoffs with respect to security, latency, and cost. For example:
+
+1. Multisig models prioritize latency and cost over security
+2. Bonding models (e.g. proof-of-stake) prioritize latency and security over cost
+3. Optimistic models prioritize security and cost over latency
+4. Natively verified models prioritize security over cost (and sometimes latency)
+
+Interchain applications necessarily adopt the tradeoffs of their interchain security models. Rather than offering a one-size-fits-all product, sovereign consensus allows applications to choose (or construct) a security model that offers the best choice of tradeoffs for that application.
+
+Furthermore, the modularity of sovereign consensus allows Hyperlane to be forward compatible with new security models as they emerge.
+
+### **Tech Spec**
+
+Sovereign consensus is powered by `Interchain Security Modules` or `ISMs`, which specify the rules that should be used when verifying an interchain message.
+
+`ISMs` must implement the folling interface:
+
+```solidity
+interface IInterchainSecurityModule {
+    /**
+     * @notice Returns an enum that represents the type of security model
+     * encoded by this ISM.
+     * @dev Relayers infer how to fetch and format metadata.
+     */
+    function moduleType() external view returns (uint8);
```

Comment:
I wonder if it would be nice for this HIP to have the canonical list for all the module types that are currently accepted as HIPs?